helpers/util.sh

```diff
@@ -264,8 +267,14 @@ buildmw() {
 
             pushd $PKG > /dev/null || die
             if [[ "$BUILDOFFLINE" = "" && "$PKG" != *"-localbuild" ]]; then
-                minfo "pulling updates..."
-                git pull >>$LOG 2>&1|| die "pulling of updates failed"
+                if [ -z "$GIT_REVISION" ]; then
+                    minfo "pulling updates..."
+                    git pull --recurse-submodules >>$LOG 2>&1|| die "pulling of updates failed"
+                else
+                    minfo "setting version to $GIT_REVISION..."
+                    git fetch >>$LOG 2>&1
+                    git reset --hard $GIT_REVISION --recurse-submodules >>$LOG 2>&1|| die "version reset failed"
```

Comment:
Please check if local changes exist before doing any changes on the git repository.
See here for an example how to check for local changes:
https://aur.archlinux.org/cgit/aur.git/tree/build_pkgs?h=linux-pf#n15